app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+include Pagy::Backend
+class UsersController < ApplicationController
+  def index
+    @pagy, @users = pagy(User.where(params[:company_id]),  items: 5)
```

Comment:
why do we need both pagy and users?

Reply:
I am using both in my index.html.erb file